ax/core/parameter_distribution.py

```diff
@@ -68,6 +71,21 @@ def distribution(self) -> rv_generic:
             )
         return dist_class(**self.distribution_parameters)
 
+    def is_environmental(self, search_space: RobustSearchSpace) -> bool:
+        r"""Check if the parameters are environmental variables of the given
+        search space.
+
+        It is sufficient to check the first parameter since the `RobustSearchSpace`
+        does not allow mixing the two types of distributions.
```

Comment:
Should we still check with `any` here to be future-proof in case we change that down the road?